Phases/Assessments/Phase_2.md

```diff
@@ -0,0 +1,38 @@
+# Phase 2 Assessment
+
+The phase 2 assessment will test your ability to write more complex JavaScript programs for the Node.js platform, to create simple HTTP applications, to write HTML & CSS to give structure and style to a web page, and to design and interact with more complex relational databases.
+
+You will _not_ be assessed on your ability to build full-stack web apps. You will be asked to build features for a front-end or back-end environment, but not both at once.
+
+## Skills Assessed
+
+- HTML & CSS (Beginner Level)
+- HTTP Applications (Beginner Level)
+- HTTP (Beginner Level)
+- Asynchronous Programming (Beginner Level)
+- Node.js (Intermediate Level)
+- Testing (Intermediate Level)
+- SQL (Intermediate Level)
+
```

Comment:
I'd like to assess the new git skills taught in this phase. they're crazy important. you cannot survive in phase 3 without these git skills.